templates/kube-vip/kube-vip.yml.j2

```diff
@@ -72,6 +72,9 @@ spec:
 {% endif %}
         image: "{{ rke2_kubevip_image }}"
         name: kube-vip
+        ports:
```

Comment:
Please make it as optional with variable like `rke2_kubevip_metrics` and disable it by default

Reply:
Why disable by default or even make it optional in the first place? kube-vip is running on `hostNetwork` and current configuration from `main` by default opens port 2112 (as per [L63](https://github.com/lablabs/ansible-role-rke2/blob/4a35717314240323038cab3dad02f2ca584e1494/templates/kube-vip/kube-vip.yml.j2#L63-L64)). This means that port is already allocated on node and `ports` section is only a no-op information to kubernetes API that such ports exists. Making this optional won't change the situation that port is open on node and only clouds that information to kubernetes API server.